.buildkite/premerge.steps.yaml

```diff
@@ -0,0 +1,30 @@
+---
+common: &common
+  agents:
+    - "agent_count=8"
+    - "boot_disk_size_gb=400"
+    - "capable_of_building=platform"
+    - "machine_type=quad"
+    - "queue=${CI_LINUX_BUILDER_QUEUE:-v4-20-06-23-173521-bk12705-2627bc3f-d}"
+    - "platform=linux"
+    - "permission_set=builder"
+    - "scaler_version=2"
+    - "environment=${CI_ENVIRONMENT:-cn-production}"
```

Comment:
Also please set `"node_stability=${CI_NODE_STABILITY:-interruptible}"` so we benefit of ~60% cost reduction.

Reply:
done